valetudo-map-card.js

```diff
@@ -431,6 +444,39 @@ class ValetudoMapCard extends HTMLElement {
       pathCtx.globalAlpha = 1;
     };
 
+    let predictedPathPoints = this.getPredictedPathPoints(mapData.attributes, mapLegacyMode);
+    if (predictedPathPoints) {
+      const pathCtx = pathCanvas.getContext("2d");
+      pathCtx.globalAlpha = this._config.path_opacity;
+
+      pathCtx.setLineDash([5,3]);
+      pathCtx.strokeStyle = pathColor;
+      pathCtx.lineWidth = this._config.path_width;
+
+      let x = 0;
+      let y = 0;
+      let first = true;
+      pathCtx.beginPath();
+      for (let i = 0; i < predictedPathPoints.length; i+=2) {
+        x = Math.floor((predictedPathPoints[i]) / widthScale) - objectLeftOffset - mapLeftOffset;
+        y = Math.floor((predictedPathPoints[i + 1]) / heightScale) - objectTopOffset - mapTopOffset;
+        if (this.isOutsideBounds(x, y, drawnMapCanvas, this._config)) continue;
+        if (first) {
+          pathCtx.moveTo(x, y);
+          first = false;
+        } else {
+          pathCtx.lineTo(x, y);
+        };
+      };
+
+      if (this._config.show_path && this._config.path_width > 0) pathCtx.stroke();
```

Comment:
Can this have an extra config option? show_predicted_path? That would allow people to turn just the predicted part off if they wanted. Still should be disabled with show_path too probably because I don't see it making sense to only show the prediction and not the current path

Reply:
Makes sense.  Added config value and tested.